config/puma.rb

```diff
@@ -7,7 +7,7 @@
 if ENV['RAILS_ENV'] == 'staging'
   threads 2, 5
 else
-  threads 2, 8
+  threads 1, ENV.fetch('RAILS_MAX_THREADS', 2).to_i
```

Comment:
Please remove the `if ENV['RAILS_ENV'] == 'staging'` branch / setup. 

There should only be one way to configure the puma server now, no custom rails env overrides, just use the RAILS_MAX_THREADS env var.

Reply:
OK, does that need a default?